setup.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright (c) 2011 SEOmoz
+# Copyright (c) 2011-2017 SEOmoz, Inc.
```

Comment:
Officially, it's SEOmoz, Inc. still, you can see this in the HTML footer on the main page.